snappiershot/plugins/tracker.py

```diff
@@ -41,6 +57,27 @@ def __init__(self, *paths: Path):
         }
         self.snapshots = self._construct()
 
+    def get_status_report(self) -> StatusReport:
+        """ Return the status report. """
+        unchecked = failed = passed = recorded = written = 0
+        for file, functions in self.snapshots.items():
+            for function_name, function_calls in functions.items():
+                for function_call in function_calls:
+                    for snapshot in function_call["snapshots"]:
+                        if snapshot == SnapshotStatus.UNCHECKED:
+                            unchecked += 1
+                        elif snapshot == SnapshotStatus.FAILED:
+                            failed += 1
+                        elif snapshot == SnapshotStatus.PASSED:
+                            passed += 1
+                        elif snapshot == SnapshotStatus.RECORDED:
+                            recorded += 1
+                        elif snapshot == SnapshotStatus.WRITTEN:
+                            written += 1
+                        else:  # pragma: no cover
+                            raise ValueError("Unknown Status")
+        return StatusReport(unchecked, failed, passed, recorded, written)
```

Comment:
I'm worried about the argument order getting mixed up here (e.g. if you switched the `recorded` and `written` inputs it wouldn't be obviously bad or fail in any way), so we should probably enforce keyword arguments only

e.g.
```python
def this_method_takes_named_args_only(*, one: int, two: str):
```
```

Reply:
I suppose, but this functionality is already tested, and I don't think named tuples supports keyword-only arguments.